pkgs/development/interpreters/php/default.nix

```diff
@@ -228,6 +232,7 @@ let
         pdo_mysqlSupport = config.php.pdo_mysql or true;
         libxml2Support = config.php.libxml2 or true;
         apxs2Support = config.php.apxs2 or (!stdenv.isDarwin);
+        embedSupport = config.php.embed or false;
```

Comment:
No. Only one SAPI can be built at a time, either "apxs2" or "embed" in this case.